### PR TITLE
Added diagnostic response

### PR DESCRIPTION
**Pull request by angmas**

### CODE REVIEW

**Comment by angmas** [comment]
5 on javascript syntax
2/3 on prototypes. I understand how to use it, but not sure how to create custom objects using the prototype function.

File: lib/diagnostic.js [hunk 1]
```diff
@@ -6,14 +6,34 @@
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
+const normalizeWords = function (words) {
+  /* assuming the const gettysburgAddress has been set up */
+  let wordArray = words.split(" ")
+  return wordArray
```
Comment (benjimelito) [inline lib/diagnostic.js:12]: On the right track, but you're missing the `toUpperCase` portion and removing punctuation.

File: lib/diagnostic.js [hunk 2]
```diff
@@ -6,14 +6,34 @@
...
 }
 
 // 2. Write a function that takes a string argument
 // and returns an array of unique normalized words.
-const uniqueWords = function (/* */) {
+const uniqueWords = function (words) {
   /* your response here */
+  let uniqueWordArray
```
Comment (benjimelito) [inline lib/diagnostic.js:19]: Nice!

**Comment by benjimelito** [comment]
Nice work, 3/4. Let me (or someone else) know if you want a deeper dive on prototypes

**Pull request closed without merging**